test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -192,7 +192,7 @@ Describe "Import-Module should be case insensitive" -Tags 'CI' {
         Import-Module testMODULE
         $m = Get-Module TESTmodule
         $m | Should BeOfType "System.Management.Automation.PSModuleInfo"
-        $m.Name | Should Be "TESTMODULE"
+        $m.Name | Should BeIn "TESTMODULE"
```

Comment:
Isn't BeIn only used for validating if the left side is contained in the collection on the right side? Don't understand why this isn't just Be?

Reply:
same as above

Reply:
Closed